src/components/views/rooms/wysiwyg_composer/utils/message.ts

```diff
@@ -71,9 +76,51 @@ export async function sendMessage(
     }*/
     PosthogAnalytics.instance.trackEvent<ComposerEvent>(posthogEvent);
 
-    const content = await createMessageContent(message, isHTML, params);
+    let content: IContent | null = null;
 
-    // TODO slash comment
+    // Functionality here approximates what can be found in SendMessageComposer.sendMessage()
```

Comment:
The logic in this block is not really new, it has been taken from the current composer's code that is run when a message is sent (can be seen in `SendMessageComposer.tsx`), with changes made where appropriate to pass correct variables to function calls etc